discord/automod.py

```diff
@@ -328,21 +328,33 @@ async def edit(
         :class:`AutoModRule`
             The updated auto moderation rule.
         """
-        transformed_actions = None
+        payload = {}
         if actions:
-            transformed_actions = [action.to_dict() for action in actions]
+            payload['actions'] = [action.to_dict() for action in actions]
+
+        if name:
+            payload['name'] = name
+
+        if event_type:
+            payload['event_type'] = event_type
+
+        if trigger:
+            payload['trigger_metadata'] = trigger.to_metadata_dict()
+
+        if enabled:
```

Comment:
Does not trigger when enabled is False. Same for exempt_roles and exempt_channels

Reply:
Works fine for me